modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/simulation/v0.1.0.json

```diff
@@ -0,0 +1,64 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/simulation/simulation/v0.1.0/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/commons/activity/v0.1.0",
+    "{{base}}/schemas/neurosciencegraph/commons/typedlabeledontologyterm/v0.1.0"
+  ],
+  "shapes": [
+    {
+      "@id": "this:DetailedCircuitSimulationShape",
+      "@type": "sh:NodeShape",
+      "label": "Model simulation activity shape",
+      "comment": "Query: Get all simulation that generates a voltage trace layer 5",
+      "nodekind": "sh:BlankNodeOrIRI",
+      "targetClass": "nsg:Simulation",
+      "and": [
+        {
+          "node": "{{base}}/schemas/neurosciencegraph/commons/activity/v0.1.0/ActivityShape"
+        },
+        {
+          "property": [
+            {
+              "path": "prov:used",
+              "name": "Model",
+              "description": "Used model.",
+              "seeAlso": "{{base}}/schemas/neurosciencegraph/simulation/modelinstance/v0.1.0/shapes/ModelInstanceShape",
+              "class": "nsg:Model",
+              "minCount": 1
+            },
+            {
+              "path": "prov:used",
```

Comment:
How can we have same paths here? "prov:used" is in both places for ModelInstanceShape and for EntityShape.

Reply:
We should use **sh:qualifiedValueShape** here (see e.g. prov:generated property shapes in tracefeatureextraction schema).

Reply:
I'm not to much in favor of having array of polymorphic items but I also don't want to change much now. So I'm happy to merge.

Reply:
sh:qualifiedValueShape is definitely needed here.

Reply:
@genric: me neither I don't like array of polymorphic items. 

The approach adopted here is more aligned with the OOP (Object Oriented Programming) approach. All items of the prov:used property share a common 'interface' (so to say) 
       => prov:Entity as type and
       => neurosciencegraph/commons/entity as shape (remember: neurosciencegraph/simulation/modelinstance extends neurosciencegraph/commons/entity).

What does it mean ?

- You can consider items in prov:used as entities as look them up as such. For example if you only want to get the attached files of each entity, you can read the distribution property. If you want to discriminate between entities, you can check the entity type. 
- Or you can consider them as of separate and different types.

This is design pattern discussion and it deserves an issue. I'll create it.